src/components/content/deployedServices/myServices/MyServices.tsx

```diff
@@ -1027,6 +1038,46 @@ function MyServices(): React.JSX.Element {
         },
     ];
 
+    const screens = useBreakpoint(true, null);
+    const getDefaultCols = () => {
+        if (!screens) {
+            return columns;
+        }
+        if (screens.xxl) {
```

Comment:
also given the nature of how responsive devices work, i think we must switch this if  loop to other direction.. from xs to xxl..